config/config.go

```diff
@@ -1091,6 +1117,8 @@ func (cfg *ConsensusConfig) SetWalFile(walFile string) {
 // ValidateBasic performs basic validation (checking param bounds, etc.) and
 // returns an error if any check fails.
 func (cfg *ConsensusConfig) ValidateBasic() error {
+	// TODO we may want to remove this check if TimeoutPropose is removed from
+	// the config
```

Comment:
Sure, created the issue https://github.com/celestiaorg/celestia-core/issues/1517

Reply:
This seems like an unnecessary TODO. If we remove TimeoutPropose or TimeoutCommit this will have to be removed because we will have a compile error. Also it's unsure whether we will remove this or not